setup.py

```diff
@@ -35,17 +37,23 @@
     'Topic :: Software Development :: Libraries',
 ]
 
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.rst").read_text()
```

Comment:
Didn't realise that you could use Path like this. :-)

Reply:
@Aiky30 Yeah I learnt this recently and quite like it 👍